crates/swc_ecma_minifier/src/program_data.rs

```diff
@@ -643,7 +643,8 @@ impl ProgramData {
             if ctx.is_op_assign {
                 e.usage_count += 1;
             } else if is_first {
-                if e.ref_count == 1
+                if e.declared
```

Comment:
@Austaras It seems like this is required, but this change causes too many regression.
Any idea?

Reply:
Being dealing with IRL issue lately. Would (hopefully) come back later next week.

Reply:
Sorry to hear that, and no worries. It's not urgent.